src/json/mod.rs

```diff
@@ -88,6 +101,19 @@ pub fn create_json(host: &AnalysisHost, crate_name: &str) -> Result<String> {
         let (ty, child_ty) = match def.kind {
             DefKind::Mod => (String::from("module"), String::from("child_modules")),
             DefKind::Struct => (String::from("struct"), String::from("child_structs")),
+            DefKind::Enum => (String::from("enum"), String::from("child_enums")),
```

Comment:
yeah, there's no real reason that we need the `child` stuff here; i was cargo-culting from modules

Reply:
this also means that this block and the one above are identical except for modules, which is annoying